shtdlib.sh

```diff
@@ -793,17 +798,20 @@ function init_nss_wrapper {
     export NSS_WRAPPER_HOSTS="${tmp_hosts_file}"
 }
 
-# Enable Software Collections, allows multiple versions of the same RPMs to be
+# Enable a Python Software Collection, SCL allows multiple versions of the same RPMs to be
 # installed at the same time.
 function enable_scl_python {
     assert [ "${os_name}" = "redhat" ]
     shopt_decorator_option_name='nounset'
     shopt_decorator_option_value='false'
     python_version="${1}"
```

Comment:
Set a default, `3.6`, or assert that `${1}` is non-empty

Reply:
Oks, I felt it was a bit redundant but you're right, people should not be surprised when running in non-strict mode